fineract-provider/build.gradle

```diff
@@ -471,21 +471,19 @@ task showTenantDBInfo {
 }
 
 
-task migrateTenantListDB {
+task migrateTenantListDB(type: org.flywaydb.gradle.task.FlywayMigrateTask) {
     description="Migrates a Tenant List DB. Optionally can pass dbName. Defaults to 'mifosplatform-tenants' (Example: -PdbName=someDBname)"
-    doLast {
-        def filePath = "filesystem:$projectDir" + '/src/main/resources/sql/migrations/list_db'
-        def tenantsDbName = 'mifosplatform-tenants';
-        if (rootProject.hasProperty("dbName")) {
-            tenantsDbName = rootProject.getProperty("dbName")
-        }
-
-        flyway.url= "jdbc:mysql:thin://localhost:3306/$tenantsDbName"
-        flyway.driver = "org.drizzle.jdbc.DrizzleDriver"
-        flyway.locations= [filePath]
-
-        flywayMigrate.execute()
+    def filePath = "filesystem:$projectDir" + '/src/main/resources/sql/migrations/list_db'
+    def tenantsDbName = 'mifosplatform-tenants';
+    if (rootProject.hasProperty("dbName")) {
+        tenantsDbName = rootProject.getProperty("dbName")
     }
+
+    flyway.url= "jdbc:mysql:thin://localhost:3306/$tenantsDbName"
+    flyway.driver = "org.drizzle.jdbc.DrizzleDriver"
+    flyway.user = 'root'
+    flyway.password = '.....'
```

Comment:
Is this password not suppose to be 'mysql'? the default password of root user. Maybe thats why the service cannot connect to the DB.

Reply:
@muarachmann pull from the latest develop commit to resolve the merge conflicts.

Reply:
okay yes i am doing so